freestyle/shared/src/test/scala/freestyle/implicits.scala

```diff
@@ -0,0 +1,35 @@
+package freestyle
+
+import cats.implicits._
+import cats.Id
+import org.scalatest.{Matchers, WordSpec}
+
+class implicitsTests extends WordSpec with Matchers {
+
+  "Implicits" should {
+
+    import algebras._
+    import freestyle.implicits._
+
+    "make monad implicitly available" in {
+      type G[A] = FreeS[SCtors1.Op, A]
+      implicitly[cats.Monad[G]].isInstanceOf[cats.Monad[G]]
```

Comment:
We could use scalatest's `compile` check here (like I used to check the availablity of implicit http4s `EntityEncoder`s in the [freestyle-http-http4s module](https://github.com/47deg/freestyle/blob/master/freestyle-http-http4s/src/test/scala/http/Http4sTest.scala#L26-L29))

```scala
import cats.Monad
"Monad[FreeS[SCtors1.Op, ?]]" should compile 
```